### ISSUE TITLE
Source map: source paths are not always relative

### ISSUE THREAD

**Issue by kottenator** [hughsk/uglifyify#46]
I use `uglifyify` with `babelify`. I do:

```sh
browserify src.js -o out.js -t [ babelify --presets=babel-preset-es2015 ] -t uglifyify -d
```

In the result source map, there is:
```json
{
  "version": 3,
  "sources": [
    "../../../usr/lib/node_modules/browserify/node_modules/browser-pack/_prelude.js", 
    "/home/usr/prj/src.js", 
    "/home/usr/prj/utils.js"
  ],
  "names": [], 
  "mappings": "...", 
  "file": "generated.js", 
  "sourceRoot": "", 
  "sourcesContent": [ "...", "...", "..."]
}
```

As you can see, `Browserify` file path is relative but my project paths are absolute. 

I don't want to disclose my file system paths. Could you fix this?

**Comment by yoshuawuyts** [comment]
ah, that's annoying. The easy solution would be to not include source maps for production (standard practice); if this - however - isn't an option for you... PR welcome!

**Comment by kottenator** [comment]
I'm not sure if I'm qualified enough to do a PR. 

Btw, when I do `browserify src.js -o out.js -t babelify -d` without `uglifyify` - it defines relative paths in the source map correctly.

**Comment by yoshuawuyts** [comment]
@kottenator ah well sure you are, :sparkles: believe in yourself :sparkles:

**Comment by rmadamanchi** [comment]
Any ETA for a release with this fix?

**Comment by yoshuawuyts** [comment]
@rmadamanchi nope, we're all super busy and this is not a priority. PRs welcome

**Comment by yoshuawuyts** [comment]
Released as `v3.0.2` :sparkles:
